### DISCUSSION TITLE
accetto/ubuntu-vnc-xfce-firefox-g3 on Cloud IDEs (GitPod, Codespaces, and Google Cloud Shell)

### DISCUSSION THREAD

**Discussion by danielmenezesbr** [accetto/ubuntu-vnc-xfce-g3#50] [General]
I have created a [repository ](https://github.com/danielmenezesbr/novnc-cloud-ide) that facilitates running the accetto/ubuntu-vnc-xfce-firefox-g3 image on cloud IDEs (GitPod, Codespaces, and Google Cloud Shell).

I use this environment to perform some tests on Firefox and download files that are later analyzed in my cloud IDE. For this reason, the [docker-compose.yml](https://github.com/danielmenezesbr/novnc-cloud-ide/blob/main/docker-compose.yml) file includes the volume "./Downloads:/home/headless/Downloads".

I am sharing this repository because it may be useful to others.

**Comment by accetto** [comment]
Hello @danielmenezesbr,

thank you for sharing this. I'm sure that It'll be useful to many.

I appreciate your contribution very much.

Regards,
accetto
